### DISCUSSION TITLE
How to disable auto reload configuration when save configuration file.

### DISCUSSION THREAD

**Discussion by BrightXiaoHan** [LunarVim/LunarVim#2478] [Q&A] [answered]
I use `autosave` plugin to autosave my buffer. But it confused me when I edit my config file. How to solve this.

**Comment by abzcoding** [comment, marked as answer]
```lua
lvim.autocommands._general_settings[4] = false
```

basically, it removes this autocommand

https://github.com/LunarVim/LunarVim/blob/10c7753d8e6f572974f9b9e0d0d8631cd13e60ea/lua/lvim/core/autocmds.lua#L25-L26

**Reply by BrightXiaoHan** [reply]
Hi, @abzcoding. Thanks for your answer. I set it up as you said.
But when I open the lvim, it always show these info
<img width="1438" alt="截屏2022-04-16 上午8 17 29" src="https://user-images.githubusercontent.com/25839309/163654402-5b202b54-61d3-40ae-a4e2-89835805ed5c.png">

**Reply by abzcoding** [reply]
what, maybe cleanup(reinstall) the autosave plugin 
cause we don't have any print that would show this
